### PR TITLE
Added version to window title.

### PR DESCRIPTION
**Pull request by Pyraqq**
Also updated the translation files for missing strings and updated the Polish translation.

### CODE REVIEW

**Comment by Pyraqq** [comment]
The version number is missing however from demo playback. I plan to fix this soon.
obraz

File: src/lobby.cpp [hunk 1]
```diff
@@ -12,8 +12,9 @@ Lobby::Lobby(AOApplication *p_ao_app) : QMainWindow()
 {
   ao_app = p_ao_app;
 
+
   //
-  this->setWindowTitle(tr("Attorney Online 2"));
+  this->setWindowTitle(tr("Attorney Online ") + ao_app->applicationVersion());
```
Comment (oldmud0) [inline src/lobby.cpp:17]:
```suggestion
  this->setWindowTitle(tr("Attorney Online %1").arg(ao_app->applicationVersion()));
```

File: src/packet_distribution.cpp [hunk 2]
```diff
@@ -268,7 +268,7 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
 
     courtroom_loaded = false;
 
-    window_title = tr("Attorney Online 2");
+    window_title = tr("Attorney Online");
```
Comment (oldmud0) [inline src/packet_distribution.cpp:271]:
```suggestion
    window_title = tr("Attorney Online %1").arg(applicationVersion());
```

File: src/packet_distribution.cpp [hunk 3]
```diff
@@ -278,7 +278,7 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
         server_name = info.name;
         server_address =
             QString("%1:%2").arg(info.ip, QString::number(info.port));
-        window_title += ": " + server_name;
+        window_title += " " + applicationVersion() + ": " + server_name;
```
Comment (oldmud0) [inline src/packet_distribution.cpp:281]:
```suggestion
        window_title += ": " + server_name;
```

**Review by oldmud0 — CHANGES_REQUESTED** [review]

**Review by oldmud0 — APPROVED** [review]

**Pull request merged**